src/main/java/net/mcreator/ui/component/JSelectableListMouseListenerWithDND.java

```diff
@@ -146,18 +145,14 @@ public void stopDNDAction() {
 			}
 
 			stopDNDAction();
+			selection = list.getSelectedIndices();
 		}
 	}
 
 	@Override public void pressFiltered(MouseEvent e, int clicks) {

```

Comment:
Do we even need a filtered mouse listener anymore given now we don't wait for 0 or 1 or 2 clicks the way it was needed before? Maybe this could be reduced to regular mouse listener

Reply:
Code search shows that this class is the only one to implement `MousePressListener`, so ideally I should remove that class too?

Reply:
Yep :)

Reply:
:heavy_check_mark: Fixed